docs/self-managed/components/orchestration-cluster/core-settings/configuration/properties.md

```diff
@@ -87,20 +87,21 @@ Disabling CSRF protection is not recommended for production environments as it l
+| `camunda.security.authentication.oidc.client-id`             | The client ID for OIDC authentication.                                                                               |                                      |
+| `camunda.security.authentication.oidc.client-secret`         | The client secret for OIDC authentication.                                                                           |                                      |
+| `camunda.security.authentication.oidc.issuer-uri`            | The issuer URI for OIDC authentication.                                                                              |                                      |
+| `camunda.security.authentication.oidc.redirect-uri`          | The redirect URI for OIDC authentication.                                                                            | `http://localhost:8080/sso-callback` |
+| `camunda.security.authentication.oidc.username-claim`        | The claim to use for the username in OIDC authentication.                                                            | `sub`                                |
+| `camunda.security.authentication.oidc.groups-claim`          | The claim to use for groups in OIDC authentication.                                                                  |                                      |
+| `camunda.security.authentication.oidc.client-id-claim`       | The claim to use for clients in OIDC authentication.                                                                 |                                      |
+| `camunda.security.authentication.oidc.audiences`             | Comma-separated list of audiences to validate in the OIDC token.                                                     |                                      |
+| `camunda.security.authentication.oidc.scope`                 | Comma-separated list of scopes to request in the OIDC token.                                                         | `openid, profile`                    |
+| `camunda.security.authentication.oidc.jwk-set-uri`           | The authorization server’s JWK Set Uri.                                                                              |                                      |
+| `camunda.security.authentication.oidc.authorization-uri`     | The authorization server’s Authorization Uri.                                                                        |                                      |
+| `camunda.security.authentication.oidc.token-uri`             | The authorization server’s Token Uri.                                                                                |                                      |
+| `camunda.security.authentication.oidc.prefer-username-claim` | Whether to prefer the username claim over the client ID claim. Default preference is client ID first, then username. | `false`                              |
```

Comment:
Order-wise, I suggest we put this below `client-id-claim` because this property directly relates to it (and username claim). In that sense, I'd also argue to make the order like this: username claim, client id claim, prefer username claim, group claim.

Lastly, I suggest we explain more specifically what this setting does, e.g.:
> Determines if a token that contains both, the configured username claim and the configured client id claim, is treated as a user or a client. If set to `true`, it is treated as a user. If set to `false`, it is treated as a client.

(the explanation of the default value is redundant with the default value column, so I wouldn't repeat that.